src/assets/js/app.js

```diff
@@ -13,5 +14,14 @@ require('foundation-sites');
 // the line below
 //import './lib/foundation-explicit-pieces';
 
+import EventContainer from './components/event-container.vue';
 
 $(document).foundation();
+
+var vue = new Vue({
+    el: '#EventContent',
```

Comment:
proč id mají první velké písmeno, je to nějaká vlastní konvence ? Resp. příjde mi to zvláštní, běžně se to nedělá...

Reply:
nikde jsem nenasel konvenci pro pojmenovavani ID, ale jinak velke 1 pismeno nepisi, tohle byl nejaky ulet a pak jsem to zkopiroval :)